content/en/docs/ibm/pipelines.md

```diff
@@ -4,11 +4,51 @@ description = "Instructions for using Kubeflow Pipelines on IBM Cloud Kubernetes
 weight = 50
 +++
 
-## Authenticating Kubeflow Pipelines with the SDK
+By default, Kubeflow pipelines on IBM Cloud are running with the Tekton backend. Below are the instructions on how to use the Kubeflow Pipelines with the Tekton backend [(kfp-tekton)](https://github.com/kubeflow/kfp-tekton).
+
+In this tutorial, we use the below single step pipeline as our example
+
+```python
+from kfp import dsl
+def echo_op():
+    return dsl.ContainerOp(
+        name='echo',
+        image='busybox',
+        command=['sh', '-c'],
+        arguments=['echo "Got scheduled"']
+    )
+
+@dsl.pipeline(
+    name='echo',
+    description='echo pipeline'
+)
+def echo_pipeline(
+):
+    echo = echo_op()
+```
+
+# Declare the Python Client for Kubeflow Pipelines
+
+## 1. Single User Kubeflow Pipelines with the SDK
+
+**Notes**:
+* Python package [`kfp-tekton`](https://pypi.org/project/kfp-tekton/) v0.4.0 or above is required.
+* This code block below is for the single user Kubeflow deployment from the manifest
+[https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_ibm.v1.2.0.yaml](https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_ibm.v1.2.0.yaml).
+
+```python
+from kfp_tekton import TektonClient
+
+KUBEFLOW_PUBLIC_ENDPOINT_URL = 'http://<Kubeflow_public_endpoint_URL>'
+KUBEFLOW_PROFILE_NAME = None
+client = TektonClient(host=KUBEFLOW_PUBLIC_ENDPOINT_URL)
+```
+
+## 2. Authenticating Multi-user Kubeflow Pipelines with the SDK
 
 **Notes**:
-* Python package `kfp` v1.0.0 is required.
-* This feature is available with multi-user, auth-enabled Kubeflow installation deployed from the manifest [https://raw.githubusercontent.com/kubeflow/manifests/v1.1-branch/kfdef/kfctl_ibm_dex_multi_user.v1.1.0.yaml](https://raw.githubusercontent.com/kubeflow/manifests/v1.1-branch/kfdef/kfctl_ibm_dex_multi_user.v1.1.0.yaml).
+* Python package [`kfp-tekton`](https://pypi.org/project/kfp-tekton/) v0.4.0 or above is required.
+* This feature is available with multi-user, auth-enabled Kubeflow installation deployed from the manifest [https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_ibm_multi_user.v1.2.0.yaml](https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_ibm_multi_user.v1.2.0.yaml).
 * Since it transports sensitive information like session cookie value over edge network, we highly recommend enabling HTTPS for the public endpoint of Kubeflow.
 
 It requires authentication via the public endpoint of Kubeflow deployment when using the Kubeflow Pipelines multi-user feature with Pipelines SDK. Below variables need to be provided, no matter coming from an in-cluster Jupyter notebook or a remote machine:
```

Comment:
I think it would be best to clarify what "It" means (the method?) in the first sentence in the beginning